onadata/apps/main/tests/test_csv_export.py

```diff
@@ -183,3 +183,110 @@ def test_csv_repeat_with_note(self):
                 "_media_all_received",
             ]
             self.assertEqual(sorted(expected_headers), sorted(actual_headers))
+
+    def test_csv_repeat_with_selects(self):
+        """
+        Test that select one and select multiple in a repeat are exported correctly.
+        """
+        md = """
+        | survey   |
+        |          |
+        |          | type                                | name               | label                            |
+        |          | text                                | name               | 1. What is your name?            |
+        |          | integer                             | age                | 2. How old are you?              |
+        |          | image                               | picture            | 3. May I take your picture?      |
+        |          | select one from yes_no              | has_children       | 4. Do you have any children?     |
```

Comment:
Should this be `select_one yes_no`?

Reply:
Both syntaxes are supported. The tutorial_w_repeat.xlsx fixture from which this form was adopted has that, and we have no failures in transforming the form to XForm.